internal/oci/image.go

```diff
@@ -182,7 +182,7 @@ func (c *LocalCache) loop(ctx context.Context) {
 					log.V(1).Info("Start pulling")
 					err := c.retryPullImage(ctx, req.ref)
 					if err != nil {
-						log.Error(err, "Error copying image")
+						log.Error(err, "Error copying oci")
```

Comment:
many error log use lowercase

```suggestion
						log.Error(err, "error copying oci")
```

Reply:
Logs should be capitalized: 
https://google.github.io/styleguide/go/decisions.html#error-strings

Reply:
but it isn't kubebuilder style:

https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v4/scaffolds/internal/templates/main.go

Reply:
controller-runtime use lower case format too:
https://github.com/kubernetes-sigs/controller-runtime/blob/76d3d0826fa9dca267c70c68c706f6de40084043/pkg/manager/server.go#L45

Reply:
if you want to change it, you can do it feel free but everywhere (other projects in ironcore too). So i recommend keep current convention for compatibility with other logs and create issue for update.

Reply:
Please open a separate a PR if you would like to change/unify the format. I will not change the format in this PR at only one single place.